wagtail_airtable/management/commands/reset_local_airtable_records.py

```diff
@@ -0,0 +1,56 @@
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = "Looks through every available model in the AIRTABLE_IMPORT_SETTINGS and unsets the `airtable_record_id`"
+
+    def _get_model_for_path(self, model_path):
+        """
+        Given an 'app_name.model_name' string, return the model class
+        """
+        app_label, model_name = model_path.split('.')
+        return ContentType.objects.get_by_natural_key(app_label, model_name).model_class()
+
+    def get_all_models(self) -> set:
```

Comment:
This logic seems to be used frequently - maybe a candidate for a utility function?

Reply:
Agreed. Will separate that functionality into a utility function. Good catch!

Reply:
Done! Thanks for the feedback! (Tests included)